charts/fluentd-elasticsearch/templates/configmaps.yaml

```diff
@@ -531,6 +531,7 @@ data:
       reload_on_failure "#{ENV['OUTPUT_RELOAD_ON_FAILURE']}"
       reload_connections "#{ENV['OUTPUT_RELOAD_CONNECTIONS']}"
       request_timeout "#{ENV['OUTPUT_REQUEST_TIMEOUT']}"
+      suppress_type_name "#{ENV['OUTPUT_SUPPRESS_TYPE_NAME']}"
```

Comment:
I'd like to see a "{{- if .Values.elasticsearch.suppressTypeName }}" here, to get sure not to change stuff for people who are updateing.